unit-tests/src/test/scala/java/util/PropertiesTest.scala

```diff
@@ -358,32 +349,51 @@ class PropertiesTest {
                  prop.store(null: OutputStream, ""))
   }
 
+  def store(props: Properties, header: String = ""): ByteArrayOutputStream = {
+    val out = new ByteArrayOutputStream()
+    props.store(out, header)
+    out.close()
+    out
+  }
+
+  def storeWriter(props: Properties,
+                  header: String = ""): ByteArrayOutputStream = {
+    val out = new ByteArrayOutputStream()
+    props.store(new OutputStreamWriter(out), header)
+    out.close()
+    out
+  }
+
+  def load(out: ByteArrayOutputStream): Properties =
+    load(new ByteArrayInputStream(out.toByteArray))
+
+  def load(in: ByteArrayInputStream): Properties = {
```

Comment:
It seems you're always calling this method with a `new ByteArrayInputStream(someBytes)`. You could take the `bytes: Array[Byte]`s directly as an argument to avoid that repetition.

Further, you could even take a `String`, since in virtually all cases you pass in `someString.getBytes()`. Speaking of which, you should use `.getBytes(StandardCharsets.ISO_8859_1)` not to depend on the system charset.

Reply:
I used this suggestion which really helped.